src/sh/scripts/use-shell

```diff
@@ -5,7 +5,7 @@ __am_prompt_use_shell() {
 	PROMPT_SHELL=${PROMPT_SHELL:-1}
 
 	if [ -z "${PROMPT_SHELL:-}" ]; then
-		print-fail "prompt: uou must specify a shell to use, either bash or zsh"
+		print-fail "prompt: you must specify a shell to use, either bash or zsh"
```

Comment:
@dmccaffery can this be called at anytime to switch which shell prompt is using and does it persist through sessions?

Reply:
yes and yes. This isn't new; it's been there since we first supported sh, bash, and zsh.

Reply:
![](https://media.giphy.com/media/83QtfwKWdmSEo/giphy.gif)